babel/src/board.js

```diff
@@ -1,214 +1,13 @@
 'use strict';
 import * as _ from 'lodash';
+import iterchunks from './tools/iterchunks';
+import evaluate as affectOrbs.evaluate from './affectOrbs/evaluate';
+import unmatch as affectOrbs.unmatch from './affectOrbs/unmatch';
+// import './affectOrbs';
```

Comment:
Yes I did both to see what is best. I want what you have there. How is it best to import it? Is `import evaluate as affectOrbs.evaluate from './affectOrbs/evaluate';` valid?